app/views/curation_concerns/generic_files/_generic_file.html.erb

```diff
@@ -1,5 +1,5 @@
 <tr class="<%= dom_class(generic_file) %> attributes">
-  <td>
+  <td class="thumbnail">
     <%= render_thumbnail_tag generic_file %>
```

Comment:
@jcoyne Just curios why we are using two separate thumbnail renders.  Would this helper do the right thing in index_default @ line 2?